bosh-stemcell/spec/support/stemcell_shared_examples.rb

```diff
@@ -37,8 +37,8 @@
       its (:stdout) { should eq('') }
     end
 
-    describe file('/etc/hosts.equiv') do
-      it { should_not be_file }
+    describe command('[ ! -e /etc/hosts.equiv ] || egrep \^[^#].+ /etc/hosts.equiv') do
```

Comment:
[STIG V38491](https://www.stigviewer.com/stig/red_hat_enterprise_linux_6/2013-06-03/finding/V-38491) mandates that `/etc/hosts.equiv` must not exist at all on the system, so although an empty file would probably not be an issue in practice, it would be in violation of that STIG's requirements. 

Should switch back to the previous behavior - ruby's [File.file?](https://ruby-doc.org/core-2.2.0/File.html#method-c-file-3F) will ensure that the file is not present, and we prefer using ruby libraries rather than shell commands where possible.